src/activities/content/ContentCompletionEntity.js

```diff
@@ -0,0 +1,34 @@
+import { Entity } from '../../es6/Entity.js';
+import { performSirenAction } from '../../es6/SirenAction.js';
+
+/**
+ * ContentCompletionEntity class representation of a d2l content completion entity.
+ */
+export class ContentCompletionEntity extends Entity {
+
+	/** @returns {bool} Whether or not the the content completion type is manual*/
+	async isContentCompletionManual() {
```

Comment:
for these 2 getter functions, there isn't really any need for these to be async, they don't need to do any awaiting or anything

Reply:
Ahh yes, you are correct, this is probably some copy pasta from when I moved this from the UCE repo in which the controller did have some async stuff, e.g.

```
	async _isContentCompletionManual() {
		const baseEntity = await this.fetchBaseEntity();
		return baseEntity?.hasClass('manual');
	}
```

I'll fix in next commit!